### ISSUE TITLE
Set property 'DeferrableContent' threw an exception

### ISSUE THREAD

**Issue by KuroThing** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#659]
This appears to be a similar issue as #597, and is related to the Flyout, but will also occur when using Default.

Versions:
MahApps.Metro: v1.5.0-alpha10
MaterialDesignThemes.MahApps: v0.0.10

Full Exception:
```
System.Windows.Markup.XamlParseException occurred
  HResult=0x80131501
  Message=Set property 'System.Windows.ResourceDictionary.DeferrableContent' threw an exception.
  Source=PresentationFramework
  StackTrace:
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)

Inner Exception 1:
FileLoadException: Could not load file or assembly 'MahApps.Metro, PublicKeyToken=f4fb5a3c4d1e5b4f' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
```

**Comment by STDGA** [comment]
Hi @ButchersBoy 
to solve this problem for me and others i have updated the Paket.lock file dependencies like this:
![image](https://user-images.githubusercontent.com/8609390/39956797-93f78400-55e7-11e8-97c0-0b2bb0a1c09c.png)

and i have created a nuget package with updated version:
https://www.nuget.org/packages/MaterialDesignThemes.MahApps.163

I hope it is ok with you, until you have time to update your nuget packages.

**Comment by tstivers1990** [comment]
@STDGA I have tried the nuget package linked. It did not resolve the issue for me.

**Comment by Keboo** [comment]
This should be resolved with the latest 4.0.0 preview releases
